docs/source/configuration/b2b.rst

```diff
@@ -183,9 +189,10 @@ Courseware Setup in edX
 
 B2B courses in edX should be set up in a particular fashion, both to make sure we can identify them within edX easily, and to allow the system to automatically create runs for new contracts.
 
-Each B2B course starts with a source course. Usually, these are separate courses and runs, but not always. If you're creating a new course, it should be created in edX with the organization ``UAI_SOURCE`` and the run tag ``SOURCE``. A corresponding course run in MITx Online should also be created. The ``import_courserun`` command can be used to help facilitate this. If you want to use an existing course, you should create a ``SOURCE`` run for it from the run you want to use as the source course in edX.
+Each B2B course starts with a source course. Usually, these are separate courses and runs, but not always. If you're creating a new course, it should be created in edX with the organization ``UAI_SOURCE`` and the run tag ``SOURCE``. A corresponding course run in MITx Online should also be created. The ``import_courserun`` command can be used to help facilitate this. If you want to use an existing course, you should create a ``SOURCE`` run for it from the run you want to use as the source course in edX. Alternatively, an existing run can be flagged as the source run in Django Admin via the ``is_source_run`` flag.
```

Comment:
This is correct but minor nit - the `SOURCE` runs should also have the flag set. The flag is the first thing the code checks for and won't be affected if we decide to change the convention for this later on.